packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -16,6 +16,7 @@ const DEFAULT_OPTIONS = {
   paddingRegular: PADDING_REGULAR,
   paddingWide: PADDING_REGULAR * 2,
   subtle: DEFAULT_SUBTLE,
+  zIndex: 0, // "z-index" for the root container of Web Chat. This will form a new stacking context so "z-index" used in children won't pollute.
```

Comment:
Does `rootZIndex` or `zIndexRoot` make more sense here?

Reply:
Renamed to `rootZIndex` to match `rootHeight`/`rootWidth` some where below.